frontend/src/static/js/components/webstatus-overview-page.ts

```diff
@@ -130,17 +136,65 @@ export class OverviewPage extends LitElement {
       .then(allFeatures => {
         // Use CELL_DEFS to define the columns and
         // get the current (active) columns.
+        const columns: string[] = [];
         const columnKeys = parseColumnsSpec(getColumnsSpec(this.location));
-        const columns = columnKeys.map(
-          columnKey => CELL_DEFS[columnKey].nameInDialog
-        );
+
+        const pushBrowserChannelName = (
+          browserColumnKey: BrowserChannelColumnKeys
+        ) => {
+          const name = CELL_DEFS[browserColumnKey].nameInDialog;
+          columns.push(name);
+
+          const browser = CELL_DEFS[browserColumnKey].options.browser!;
+          const browserLabel = BROWSER_ID_TO_LABEL[browser];
+          const channel = CELL_DEFS[browserColumnKey].options.channel!;
+          const channelLabel = CHANNEL_ID_TO_LABEL[channel];
+
+          const wptName = `${browserLabel} WPT ${channelLabel} Score`;
+          columns.push(wptName);
+        };
+
+        columnKeys.forEach(columnKey => {
+          const name = CELL_DEFS[columnKey].nameInDialog;
+          switch (columnKey) {
+            case ColumnKey.Name:
+              columns.push(name);
+              break;
+            case ColumnKey.BaselineStatus:
+              columns.push(name);
+              break;
+            case ColumnKey.StableChrome:
+            case ColumnKey.StableEdge:
+            case ColumnKey.StableFirefox:
+            case ColumnKey.StableSafari:
+            case ColumnKey.ExpChrome:
+            case ColumnKey.ExpEdge:
+            case ColumnKey.ExpFirefox:
+            case ColumnKey.ExpSafari:
+              pushBrowserChannelName(columnKey);
+              break;
+          }
+        });
 
         // Convert array of feature rows into array of arrays of strings,
         // in the same order as columns.
         const rows = allFeatures.map(feature => {
           const baselineStatus = feature.baseline?.status || '';
           const browserImpl = feature.browser_implementations!;
-          const row = [];
+          const wptData = feature.wpt;
+          const row: string[] = [];
+
+          const pushBrowserChannelValue = (
+            browserColumnKey: BrowserChannelColumnKeys
+          ) => {
+            const browser = CELL_DEFS[browserColumnKey].options.browser!;
```

Comment:
I think we can use some generic arguments to prevent the use of `!`.

Reply:
We would have to narrow the type of options to correspond to the BrowserChannelColumnKeys. Not sure TypeScript could infer that on its own.

Reply:
If it can't infer, check if the values are undefined first and throw an error if so.

Reply:
Done.  Added a utility to webstatus-overview-cells for this.